### PR TITLE
[sc-4029] program change for player

### PR DESCRIPTION
**Pull request by StephanieZuniga**
- Add support for program change
- Add test in example project that can be commented out and we can make use of it when implementing UITests

### CODE REVIEW

File: apps/DemoApp/DemoApp/PlayerViewController.swift [hunk 1]
```diff
@@ -154,7 +164,7 @@ class PlayerViewController: UIViewController {
     }
     
     func setupMUX() {
-        let playerData = MUXSDKCustomerPlayerData(environmentKey: "YOUR_ENV_KEY_HERE")
+        let playerData = MUXSDKCustomerPlayerData(environmentKey: "shqcbkagevf0r4jh9joir48kp")
```
Comment (bmulligan-kg) [inline apps/DemoApp/DemoApp/PlayerViewController.swift:167]: is this your key

File: apps/DemoApp/DemoApp/PlayerViewController.swift [hunk 2]
```diff
@@ -222,6 +232,38 @@ class PlayerViewController: UIViewController {
         MUXSDKStats.videoChangeForPlayer(name: self.playerName, customerData: customerData)
     }
     
+    @objc func MUXProgramChange() {
+        let playerData = MUXSDKCustomerPlayerData(environmentKey: "shqcbkagevf0r4jh9joir48kp")
```
Comment (bmulligan-kg) [inline apps/DemoApp/DemoApp/PlayerViewController.swift:236]: and this?
i prefer to just have a single var to store the env key in and reference it :P

File: apps/DemoApp/DemoApp/PlayerViewController.swift (hunk 1 above)
Comment (StephanieZuniga) [reply apps/DemoApp/DemoApp/PlayerViewController.swift]: updated

File: apps/DemoApp/DemoApp/PlayerViewController.swift (hunk 2 above)
Comment (StephanieZuniga) [reply apps/DemoApp/DemoApp/PlayerViewController.swift]: yeah much better, I've updated.

File: MUXSDKKaltura/MUXSDKKaltura/MUXSDKStats.swift [hunk 3]
```diff
@@ -117,6 +117,24 @@ public class MUXSDKStats: NSObject {
         bindingsManager.customerDataStore.updateData(customerData, forPlayerName: name)
     }
     
+    /**
+     Signals that a player is now playing a different video of a playlist; or a different program of a live stream.
+     
+     Use this method to signal that the player is now playing a different video of a playlist, or a different program of a live stream. The player name provided must have been passed as the name in a monitorPlayer:withPlayerName:andConfig: call. The config provided should match the specifications in the Mux docs at https://docs.mux.com and should include all desired keys, not just those keys that are specific to this video. If the name of the player provided was not previously initialized, an exception will be raised.
```
[the remark this answers is not in the document]
Comment (nidhik) [reply MUXSDKKaltura/MUXSDKKaltura/MUXSDKStats.swift:123]: Perhaps this:
```
Use this method to signal that the player is now playing a different video of a playlist, or a different program of a live stream. The player name provided must have been passed as the name in a monitorPlayer:withPlayerName:andConfig: call. The config provided should match the specifications in the Mux docs at https://docs.mux.com and should include all desired keys, not just those keys that are specific to this video. If the name of the player provided was not previously initialized, a warning will be logged and this call will have no effect.
```

File: MUXSDKKaltura/MUXSDKKaltura/MUXSDKStats.swift [hunk 4]
```diff
@@ -117,6 +117,24 @@ public class MUXSDKStats: NSObject {
...
+     
+     - Parameters:
+        - name: The name of the player to update
+        - customerData: A MUXSDKCustomerData object with player, video, and view metadata
+     */
+    public static func programChangeForPlayer(name: String, customerData: MUXSDKCustomerData) {
+        guard let binding = bindingsManager.bindings[name] else {
+            return
```
Comment (nidhik) [inline MUXSDKKaltura/MUXSDKKaltura/MUXSDKStats.swift:131]: let's log a warning here?

**Review by nidhik — APPROVED** [review]

**Comment by nidhik** [comment]
lgtm, pending some copy changes and log statement

**Review by nidhik — APPROVED** [review]

**Pull request merged**